### PR TITLE
Swap - additional errors (part 2)

### PR DESCRIPTION
**Pull request by roknovak**

### CODE REVIEW

File: brd-android/fabriik-common/src/main/java/com/fabriik/common/data/model/Profile.kt [hunk 1]
```diff
@@ -54,6 +54,13 @@ fun Profile?.availableDailyLimit(): BigDecimal = this?.exchangeLimits?.available
 
 fun Profile?.availableLifetimeLimit(): BigDecimal = this?.exchangeLimits?.availableLifetime() ?: BigDecimal.ZERO
 
+fun Profile?.kyc2ExchangeLimit(): BigDecimal? {
```
Comment (dabitdev) [inline brd-android/fabriik-common/src/main/java/com/fabriik/common/data/model/Profile.kt:57]: All this methods consider the possibility of the this object to be null. Why is that required?

Comment (roknovak) [reply brd-android/fabriik-common/src/main/java/com/fabriik/common/data/model/Profile.kt:57]: Profile can be null if user is not registered. I added extension functions to nullable Profile object to avoid setting default value everytime these methods are called.

Comment (dabitdev) [reply brd-android/fabriik-common/src/main/java/com/fabriik/common/data/model/Profile.kt:57]: fair enough

**Review by dabitdev — APPROVED** [review]

**Review by lrupnikk — APPROVED** [review]
